python/Ganga/new_tests/GPI/Parser/TestSelect.py

```diff
@@ -2,47 +2,68 @@
 
 from ..GangaUnitTest import GangaUnitTest
 
+job_num = 5
+job_names = ['a', 'b', 'c', 'd', 'e']
+
 class TestSelect(GangaUnitTest):
 
-    def test_SelectJob(self):
-        
-        from Ganga.GPI import Job, jobs
-        j=Job()
+    def setUp(self):
+        """Make sure that the Job object isn't destroyed between tests"""
+        super(TestSelect, self).setUp()
+        from Ganga.Utility.Config import setConfigOption
+        setConfigOption('TestingFramework', 'AutoCleanup', 'False')
```

Comment:
I would but the exact content of the objects varies at different stages depending on things like flush operations to disk. This is difficult and short of turning all of the auto-flushing off, manually putting in a flush op, testing things before and after the flush op that the select works as expected. And there we have PR #372 come into play.

This can be done, but is potentially overkill for just checking what is checked here.

I'm not sure that this is the best motivation to push for cleanly un-loading objects from the repo. (I think #374 is the best argument for having a clean load/unload mechanism) but yes having a load/unload would be nice, just getting far out of scope of the PR.